pubspec.yaml

```diff
@@ -1,28 +1,42 @@
 name: grpc
 description: Dart implementation of gRPC, a high performance, open-source universal RPC framework.
 
-version: 2.9.0
+version: 3.0.0-nullsafety.0
 
 homepage: https://github.com/dart-lang/grpc-dart
 
 environment:
-  sdk: '>=2.8.0 <3.0.0'
+  sdk: '>=2.12.0-0 <3.0.0'
```

Comment:
This change seems to have broken the grpc/grpc interop test.  See the error message in https://github.com/grpc/grpc/pull/25305.

In particular, the [interop/pubspec](https://github.com/grpc/grpc-dart/blob/master/interop/pubspec.yaml#L13) uses this pubspec at the root level for the grpc package. On the grpc side, our dart interop base image is still using `google/dart:2.10`.

However, if I attempt to use `google/dart:2.12-beta`, I ran into the following error:

```
2021-01-29 12:30:30,244 ../lib/src/client/common.dart:64:19: Error: The argument type 'R Function(R?)' can't be assigned to the parameter type 'FutureOr<R> Function(R?)' because 'R' is nullable and 'R' isn't.
            .then(_ensureOneResponse));
                  ^

2021-01-29 12:30:30,244 FAILED: cloud_to_prod:dart:default:empty_unary:tls [ret=254, pid=27507, time=4.2sec]
```